internal/characters/travelergeo/abil.go

```diff
@@ -21,22 +21,22 @@ func (c *char) Attack(p map[string]int) (int, int) {
 		Mult:       attack[c.NormalCounter][c.TalentLvlAttack()],
 	}
 
-	c.Core.Combat.QueueAttack(ai, core.NewDefCircHit(0.3, false, core.TargettableEnemy), 0, f-1)
+	c.Core.Combat.QueueAttack(ai, core.NewDefCircHit(0.3, false, core.TargettableEnemy), f-1, f-1)
 
 	if c.NormalCounter == c.NormalHitNum-1 {
 		//add 60% as geo dmg
 		ai := core.AttackInfo{
 			ActorIndex: c.Index,
 			Abil:       "A2",
-			AttackTag:  core.AttackTagNone,
```

Comment:
Are you sure about this one? I thought I was right on this one having no attack tag. There was a ticket in KQM on this I think?

Reply:
its in KQM TCL

https://library.keqingmains.com/characters/geo/traveler-geo#frenzied-rockslide
> This damage bonus is also considered Normal Attack Damage that also deals Elemental DMG.